apps/rotm/views/can-we-contact.html

```diff
@@ -1,11 +1,14 @@
 {{<partials-page}}
-  {{$page-content}}
-    <p>Your details will be shared with the Counter-Terrorism Internet Referral Unit in case they need more information. Your details will be kept confidential and won’t be used for any purposes other than those stated in the <a href="https://www.gov.uk/help/privacy-policy">privacy policy</a>.
-  	</p>
+{{$page-content}}
+<p>Your details will be shared with the Counter-Terrorism Internet Referral Unit in case they need more information.
+  Your details will be kept confidential and won’t be used for any purposes other than those stated in the <a
+    class="govuk-link"
```

Comment:
I don't think you need to add the class here as it is being applied to <a> tags via the css file